liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojoSupport.java

```diff
@@ -138,22 +138,51 @@ protected void installLooseConfigWar(MavenProject proj, LooseConfigData config,
             setLooseProjectRootForContainer(proj, config);
         }
 
-        LooseWarApplication looseWar = new LooseWarApplication(proj, config);
-        looseWar.addSourceDir(proj);
-        looseWar.addOutputDir(looseWar.getDocumentRoot(), new File(proj.getBuild().getOutputDirectory()),
-                "/WEB-INF/classes");
-
-        // retrieve the directories defined as resources in the maven war plugin
-        Map<String,String> webResources = MavenProjectUtil.getWebResourcesConfiguration(proj);
-        if (webResources != null) {
-            for (String directory : webResources.keySet()) {
-                String targetPath = webResources.get(directory)==null ? "/" : "/"+webResources.get(directory);
-                looseWar.addOutputDir(looseWar.getDocumentRoot(), new File(proj.getBasedir().getAbsolutePath(), directory), targetPath);
+        LooseWarApplication looseWar = new LooseWarApplication(proj, config, log);
+
+        if (looseWar.isExploded()) {
+
+            // If I'm filtering web.xml, etc., I want to monitor from the exploded dir, not via a source dir
+            if (!looseWar.isFilteringDeploymentDescriptors()) {
+                looseWar.addSourceDir();
```

Comment:
What does this `addSourceDir()` do when there is no parameter? Before the `proj` was passed in on this call. I also see further down in the `else` statement that no parameter is passed.

Reply:
Since we are passing the `project` to the LooseWarApplication constructor above this, we just use that rather than passing it in on the method call.